pyproject.toml

```diff
@@ -124,6 +125,9 @@ enable_error_code = [
     "redundant-expr",
     "truthy-bool",
 ]
+namespace_packages = true
+mypy_path = "src"
```

Comment:
I'd move mypy_path to be next to files, and put namespace_packages and explicit_package_bases right below it. After that, LGTM.

Reply:
Force-pushed https://github.com/python-poetry/poetry/commit/4e32bade3c76fca133f7a82795f686dedc3bc679 with these changes.